action/src/main.ts

```diff
@@ -163,7 +163,9 @@ class Inputs {
           this.arch = "android_armv7";
         }
       } else if (this.host === "windows") {
-        if (compareVersions(this.version, ">=", "5.15.0")) {
+        if (compareVersions(this.version, ">=", "6.8.0")) {
+          this.arch = "win64_msvc2022_64";
+        } else if (compareVersions(this.version, ">=", "5.15.0")) {
```

Comment:
```suggestion
        if (compareVersions(this.version, ">=", "6.7.3")) {
          this.arch = "win64_msvc2022_64";
        } else if (compareVersions(this.version, ">=", "5.15.0")) {
```

Per https://github.com/jurplel/install-qt-action/pull/251#issuecomment-2376239741 .

Hi, @xavier2k6 ! Do you happen to know the URL to the official post about the MSVC version upgrade since 6.7.3?

Reply:
>In one of the next bug-fix releases of Qt 6.7, possibly in Qt 6.7.2, we will add packages built with MSVC 2022 in addition to the MSVC 2019 ones as we have now. The upcoming Qt 6.8 will have packages for Windows built with MSVC 2022 only, and MSVC 2019 ones will be discontinued in binary packages. Future Qt 6.7.x bug-fix releases will still contain builds wirh MSVC2019 in binary packages as long as Qt 6.7 is supported.

https://www.qt.io/blog/moving-to-msvc-2022-in-qt-68

Reply:
So 6.7.3 still defaults to MSVC 2019.

Reply:
I checked the Qt Online Installer and It doesn't offer any option to Install Qt6 6.7.3 for MSVC-2022

Reply:
FWIW, the MSVC-2022 builds are available from the online installer's mirrors (the some ones that aqt pulls from, and this install-qt-action uses) here: https://download.qt.io/online/qtsdkrepository/windows_x86/desktop/qt6_673_msvc2022/

Reply:
but the default still MSVC-2019, those who want MSVC-2022 could specify it.